### ISSUE TITLE
[Bug] Pinging changing case of player's name.

### ISSUE THREAD

**Issue by codebycam** [Draycia/Chat#187]
With pinging a player in chat, if the player's name is mentioned with nothing else in the message it sets the name to lowercase (unintended), but if the player is mentioned with other words, the case stays the same as the player's username. 

![rgqgmvN](https://user-images.githubusercontent.com/41496198/147902848-0f3154b2-f755-4183-8388-94fbb8f89b5f.png)

Running 2.0.6 with the following ping settings:
```
ping-settings {
    # The color your name will be when another player mentions you.
    highlight-text-color=yellow
    name="minecraft:block.note_block.cow_bell"
    pitch=0.9
    play-sound=true
    prefix=""
    source=master
    volume=1
}
```

Have tested this with and without a `prefix` for the ping, with the results being the same.

**Comment by Draycia** [comment]
https://github.com/Hexaoxide/Carbon/blob/5540c7d813662820cc3306766453f89bc3fd28ff/common/src/main/java/net/draycia/carbon/common/listeners/PingHandler.java#L54-L62

PIng replace method for future reference.

**Comment by Draycia** [comment]
fixed in da6a9f1293c95da8dc4f338cbc8f8b0081a08d9d
